src/snippets/conda-install-full.rst

```diff
@@ -0,0 +1,5 @@
+.. code-block:: bash
+
+   mamba create -n nextstrain \
+     -c conda-forge -c bioconda -c defaults --strict-channel-priority --yes \
+     nextstrain-cli augur auspice nextalign nextclade snakemake git epiweeks pangolin pangolearn
```

Comment:
I'm all in favor for less duplication, but this won't work in Anaconda Prompt (cmd or PS) where the line continuation char is `` ` `` instead of `\`.

Suggestion: rename this file to `src/snippets/conda-install-full-bash.rst` and have another file `src/snippets/conda-install-full-powershell.rst` with contents:

```rst
.. code-block:: powershell

   mamba create -n nextstrain `
     -c conda-forge -c bioconda -c defaults --strict-channel-priority --yes `
     nextstrain-cli augur auspice nextalign nextclade snakemake git epiweeks pangolin pangolearn
```

(same for `conda-install-minimal.rst`)

Reply:
Ah, well, that's just the code that was there. But I can fix it in this PR too.

Reply:
Ah, nevermind, I indeed missed where it went from `.. code-block:: none` to `.. code-block:: bash`!

Reply:
Oh, actually this suggestion would just be for `conda-install-minimal.rst` since the full install for Native isn't a valid installation path for Windows.

Reply:
Done in ea1af71.